scripts/core/kernel.yml

```diff
@@ -215,7 +219,7 @@ params:
       desc: "[in] size of the Kernel SubGroup property value"
     - type: "void*"
       name: pPropValue
-      desc: "[in,out][range(0, propSize)][optional] value of the Kernel SubGroup property."
```

Comment:
shouldn't you keep the range here? `range(0, propSize)` is not equivalent to  `typename(propName, propSize)`, right?

Reply:
This is just a pointer to a byte array of type `void*`. This is true of all pointers, but we don't mark, e.g., `size_t*` as range(0, 8) :) Also, you can't use index operator on 'void *', so this just creates issues in templates.
This range qualifier is useless, as far as I can tell.

Reply:
Ok, that makes sense, I didn't fully get what range() was supposed to mean